licenses/android-googletv-license

```diff
@@ -0,0 +1,2 @@
+
+601085b94cd77f0b54ff86406957099ebe79c4d6
```

Comment:
Hey, the other changes look great!

Can you tell me about these licenses file? I dont understand their purpose.
Also, you made a similar change for travis file also.

Reply:
Is this change to make SDK accept the licenses?

Can we rathar use this command: https://stackoverflow.com/questions/38096225/automatically-accept-all-sdk-licences

Reply:
Yes, you got it right. This change is to make SDK accept the license.

**What was the problem?**
Travis CI was unable to make SDK auto accept the licenses.

**What this change did?**
In this change I added license files ( in the license folder ) from my local directory. And in .travis.yml file edit, told Travis CI to use given license files if not accepting automatically.

**But why this change and not your mentioned simple command?**
Well, I have gone through various trials including your mentioned one, but none of them worked expect this one.

Reply:
In commit df76b8dafb037070c9adf07c56af4e0fdcc09821, removed unnecessary license files and removed unnecessary commands from travis.yml file.

Reply:
I decided to solve the problem in your mentioned way (because it is simple, short & sweet). Although that command didn't work in the past, this time, with a tiny modification, it worked. :)

Thanks for reminding me that command.

Reply:
Great! This is awesome. Thanks for spending so much time in fixing this :)

Reply:
Your welcome and thank you for your amazing response. It could never had done without your quick response. 👍

**I am working further to make UI better...**

Happy coding!